liqd_product/templates/header.html

```diff
@@ -1,20 +1,20 @@
 {% load userindicator static i18n thumbnail %}
 
 <header class="main-header" aria-label="{% trans 'Header' %}">
-    <div class="main-header__border-top"></div>
+    <div class="main-header__border-top {% if not PARTNER %}main-header__border-top--platform{% endif %}"></div>
```

Comment:
just a small note you may also use
`{{ PARTNER|yesno:",main-header__border-top--platform" }}` instead of the if. although i am not sure at all if that is more readable ;)